less/v2/chat.less

```diff
@@ -605,6 +605,48 @@
   .visible-dc { display: contents; }
 }
 
+/* ---------------------------------- */
+/*  Enchantment                       */
+/* ---------------------------------- */
+
+enchantment-application {
+  .drop-area {
+    display: grid;
+    gap: 4px;
+    min-block-size: 40px;
+    margin: 4px;
+    border: 1px dashed var(--dnd5e-color-crimson);
+    border-radius: 4px;
+    padding: 4px;
+    background: var(--dnd5e-color-card);
+
+    p {
+      place-content: center;
+      text-align: center;
+    }
+
+    .preview {
+      display: flex;
+      align-items: center;
+      gap: 6px;
```

Comment:
Perhaps some slight separation between the rows:

![image](https://github.com/foundryvtt/dnd5e/assets/207433/f65f5cdd-ea4d-4d38-9f1b-985a77fc9fc8)

```css
.preview:not(:last-child) {
  border-bottom: 1px dotted var(--color-border-light-2);
  padding-bottom: 4px;
}
```